src/java/org/apache/cassandra/net/AsyncStreamingOutputPlus.java

```diff
@@ -53,6 +54,8 @@
 {
     private static final Logger logger = LoggerFactory.getLogger(AsyncStreamingOutputPlus.class);
 
+    private static final BufferPool bufferPool = BufferPoolManager.temporary();
```

Comment:
nit: If we ever want to mock out the pool for testing, we could make it an instance field, pass into the constructor, etc. (I suppose we don't carry around so many references this way.)

Reply:
I will change it to instance field.. we can mock it when tests need it.